test/remote/custom_rules.js

```diff
@@ -0,0 +1,135 @@
+/* Any copyright is dedicated to the Public Domain.
+ * http://creativecommons.org/publicdomain/zero/1.0/ */
```

Comment:
A tiny nit: I think it would be clearer if this file were named "user_defined_rules.js" to match the name of the config item it's testing.  "Custom" is too close to "customs" and could cause confusion.